### PR TITLE
enhancement: addressbook users list and details view implemented

### PR DESCRIPTION
**Pull request by Daljit-fleetHawks**
## Change Description

I have implemented the details and list of remaining users except Carriers because needs to confirm outer carriers and onBoards carrier fields.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by kunalfleethawks — APPROVED** [review]
Looks good. @amritfleethawks  please review

File: src/app/shared/popups/address-book/address-book.component.ts [hunk 1]
```diff
@@ -266,11 +268,10 @@ export class AddressBookComponent implements OnInit {
           this.receivers = receivers.Items,
           this.staffs = staffs.Items,
           this.fcCompanies = fcCompanies.Items,
-
           this.allData = [...this.customers, ...this.drivers, ...this.brokers, ...this.vendors,
                             ...this.carriers, ...this.shippers, ...this.receivers, ...this.staffs, ...this.fcCompanies];                           
-
-          this.countries = countries;
+          this.countries = countries.Items;
+          console.log("shippers", this.shippers);
```
Comment (amritfleethawks) [inline src/app/shared/popups/address-book/address-book.component.ts]: please remove this line...

File: src/app/shared/popups/address-book/address-book.component.ts [hunk 2]
```diff
@@ -291,6 +292,13 @@ export class AddressBookComponent implements OnInit {
     });
   }
 
+  openDetail(targetModal, data) {
+    $('.modal').modal('hide');
+    this.userDetailTitle = data.firstName;
+    const modalRef = this.modalService.open(targetModal);
+    this.userDetailData = data;
+    console.log("this.userDetailData", this.userDetailData);
```
Comment (amritfleethawks) [inline src/app/shared/popups/address-book/address-book.component.ts:299]: please remove this line...

**Review by Daljit-fleetHawks — COMMENTED** [review]
changes done

**Review by amritfleethawks — APPROVED** [review]

**Pull request merged**